### DISCUSSION TITLE
Welcome to FinancePy Discussions!

### DISCUSSION THREAD

**Discussion by domokane** [domokane/FinancePy#104] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by paolobaroni** [comment]
is there a way to build yiled curves like a mixture of dep / FRA /Swap for example or OIS?
Thanks very interesting project!

**Comment by domokane** [comment]
Thanks. Yes. Check example notebooks. 

https://github.com/domokane/FinancePy/blob/master/notebooks/products/rates/FINIBORDUALCURVE_BuildingUSDDualCurveSwapValuationQLExampleDeriscope.ipynb

**Comment by paolobaroni** [comment]
This is huge mate, congratulation!
I normally use quantlib for pricing but in the next few days I will try
FinancePy lib!
P

Il giorno lun 9 gen 2023 alle ore 17:10 domokane ***@***.***>
ha scritto:

**Comment by domokane** [comment]
<!--
/* Font Definitions */
@font-face
	{font-family:"Cambria Math";
	panose-1:2 4 5 3 5 4 6 3 2 4;}
@font-face
	{font-family:Calibri;
	panose-1:2 15 5 2 2 2 4 3 2 4;}
/* Style Definitions */
p.MsoNormal, li.MsoNormal, div.MsoNormal
	{margin:0cm;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
a:link, span.MsoHyperlink
	{mso-style-priority:99;
	color:blue;
	text-decoration:underline;}
.MsoChpDefault
	{mso-style-type:export-only;}
@page WordSection1
	{size:612.0pt 792.0pt;
	margin:72.0pt 72.0pt 72.0pt 72.0pt;}
div.WordSection1
	{page:WordSection1;}
